src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -392,18 +395,22 @@ private static void ReplaceExceptionHandlerBoundary(ExceptionHandler handler, In
                 handler.TryStart = newTarget;
         }
 
-        private static bool IsExcludeAttribute(CustomAttribute customAttribute)
+        private bool IsExcludeAttribute(CustomAttribute customAttribute)
         {
-            var excludeAttributeNames = new[]
+            // The default custom attributes used to exclude from coverage.
+            IEnumerable<string> excludeAttributeNames = new List<string>()
             {
                 nameof(ExcludeFromCoverageAttribute),
-                "ExcludeFromCoverage",
-                nameof(ExcludeFromCodeCoverageAttribute),
-                "ExcludeFromCodeCoverage"
+                nameof(ExcludeFromCodeCoverageAttribute)
```

Comment:
This is a backward incompatible change. We should make specifying the `Attribute` suffix optional

Reply:
Is this actually incompatible? From what I saw of the usage even if you use the short name attribute like `[ExcludeFromCoverage]` the attribute name still comes across with the full name `ExcludeFromCoverageAttribute`. Were you trying to account for people writing custom attributes to handle excluding from coverage that were named the same as your attributes?

Reply:
It's more of a user experience thing. Since the compiler doesn't enforce the `Attribute` suffix I don't want us to as well. I want the user to be able to specify `Obsolete` and still be fine. You can simply append `Attribute` to the name if it's not specified